python/cudf/bindings/cudf_cpp.pyx

```diff
@@ -127,6 +127,7 @@ cdef get_scalar_value(gdf_scalar scalar):
         GDF_INT32:   scalar.data.si32,
         GDF_INT16:   scalar.data.si16,
         GDF_INT8:    scalar.data.si08,
+        GDF_BOOL8:   scalar.data.b08,
         GDF_BOOL8:   np.array(scalar.data.b08).astype(np.bool_),
```

Comment:
You now have duplicate dictionary keys here, I’m guessing this line should be removed to just use the `scalar.data.b08`